.github/workflows/moodle-ci.yml

```diff
@@ -49,7 +49,7 @@ jobs:
       fail-fast: false
       matrix:
         php: ['7.1','7.4']
-        moodle-branch: ['MOODLE_35_STABLE', 'MOODLE_36_STABLE', 'MOODLE_37_STABLE', 'MOODLE_38_STABLE', 'MOODLE_39_STABLE', 'MOODLE_310_STABLE', 'MOODLE_311_STABLE', 'master']
```

Comment:
We DON'T want tests to pass on `master` anymore?

Reply:
We added branch `master` on the matrix a month ago to precheck the behat tests before moodle release `MOODLE_400_STABLE`. It's a good idea to keep the `master` branch for future versions.